packages/contracts/src/address-names.ts

```diff
@@ -0,0 +1,31 @@
+type addrNames = {
+  [key: string]: string
+}
+
+/**
+ * For contracts and accounts used in the Address Manager, this object defines the correct names to be used in
+ * the Address Manager and deployment artifacts.
+ */
+export const managedNames: { [key: string]: addrNames } = {
+  deployedContracts: {
```

Comment:
Stylistic discussion:

Could we do something like:

```
const names = {
  managed: {
     contracts: {
       OVM_BondManager: ...,
       OVM_L1CrossDomainMessenger: ...
       Proxy__OVM_L1CrossDomainMessenger: ...,
     },
     accounts: {
        ...
     }
  },
  unmanaged: {
    ...
  },
}
```

Note using the same naming format as with the `predeploys`.

Reply:
OK, I did that. I used two commits: 
1. https://github.com/ethereum-optimism/optimism/pull/1695/commits/cfc8b9b4e0277a55459d156a46fd9417a7cd8b48: moves everything into a singled nested object.

2. [`88312e2` (#1695)](https://github.com/ethereum-optimism/optimism/pull/1695/commits/88312e24a7db20276e45dcb2a3b1ecc01d96b2bc): changes the names to use the `predeploy.ts` formatting. 

TBH I'm not a fan of the change in \#2, it mean we still sling around these inconsistent prefixes. But I don't want to bikeshed, so it's included here.